### PR TITLE
Remote db connection

### PR DESCRIPTION
**Pull request by elizabethcan**
This branch changes the db config to use my remote db instead of local

### CODE REVIEW

File: data/index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 const { Pool } = require('pg');
 
 const pool = new Pool({
-  // user: '',
-  host: 'localhost',
+  user: 'postgres',
+  host: '13.57.51.215',
```
Comment (pon-y) [inline data/index.js]: Please remove credentials from github

**Review by pon-y — CHANGES_REQUESTED** [review]
Please remove credentials from github

File: data/example.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports.example = {
```
Comment (pon-y) [inline data/example.config.js:1]: Nice. Thanks!

**Review by pon-y — APPROVED** [review]
Nice. Thanks!

**Review by PlasmaIntec — COMMENTED** [review]
Approved

**Pull request merged**